hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -562,6 +594,20 @@ void _imxrt_init(void)
 	if (*(imxrt_common.stk + stk_ctrl) & 1)
 		*(imxrt_common.stk + stk_ctrl) &= ~1;
 
+	/* Disable deep sleep */
+	*(imxrt_common.scb + scb_scr) &= ~(1 << 2);
+	imxrt_dataBarrier();
+
+	/* HACK - temporary fix for crash when booting from FlexSPI */
+	_imxrt_setDirectLPCG(pctl_lpcg_semc, 0);
+
+	/* Disable USB cache (set by bootrom) */
```

Comment:
Isn't it needed for USB device controller?

Reply:
This is something that bootrom set and haven't cleaned after itself. It is set after boot from FlexSPI, but reset after boot from serial downloader. It's done in NXP's SDK.